schedule/yast/qam-yast_self_update.yaml

```diff
@@ -32,13 +31,6 @@ schedule:
   - installation/first_boot
   - '{{efi}}'
 conditional_schedule:
-  sles_product:
-    VERSION:
-      15-SP3:
-        - installation/module_registration/add_ltss_regcode
-      15-SP4:
-        - installation/module_registration/add_additional_regcodes
-        - installation/module_registration/import_untrusted_gnpupg_key
```

Comment:
looks like we are missing the deletion of this now unused files, right?

Reply:
and tests/installation/module_registration/import_untrusted_gnpupg_key.pm ?

Reply:
As 15-SP4 has 'we' addon, so it need import_untrusted_gnpupg_key.
At 15-SP3, the 'we' addon has expired.

Reply:
are the 3 test modules above used somewhere else? if not, we need to clean up.